source/includes/markdown/_13_1-scim.html.md

```diff
@@ -31,10 +54,23 @@ The accepted user attributes are:
 |name.givenName|string|Unsupported, use `formatted`|
 |name.familyName|string|Unsupported, use `formatted`|
 |name.formatted|string|The full name of the user.|
-|emails|complex|Email addresses for the user.|
+|emails|multi-valued complex|Email addresses for the user.|
 |emails.value|string|Email address for the user|
 |email.primary|string|Whether this email address is the preferred email address for this user. `true` may only appear once for this  attribute.|
 |active|boolean|Indicates whether the user's account is active in Asana.|
 |photo|string|URL for the user's profile photo.|
+|title|string|The user's title, such as "Vice President".|
+|preferredLanguage|string|The user's preferred language. Used for selecting the localized user interface.|
+|"urn:ietf:params:scim:<br>schemas:extension:enterprise:<br>2.0:User"|complex|The attribute definition.|
+|"urn:ietf:params:scim:<br>schemas:extension:enterprise:<br>2.0:User.department"|string|The department the user belongs to.|
+
+
+The accepted group attributes are:
+
+|Attribute|Type|Info|
+|---------|----|----|
+|displayName|string|Unique identifier for the team. REQUIRED.|
+|members|multi-valued complex|The members of a team.|
+|members.value|string|The team member's user ID.|
 
```

Comment:
Generally, I find these docs extremely dry [more than the rfc]. I think we should add a few examples. Thoughts?

Reply:
Yes! We should add some examples like the ones for [Errors](https://developers.asana.com/docs/errors) with Request and Response!! Will do that!!